text/0000-add-contributer-graph.md

```diff
@@ -0,0 +1,71 @@
+# Meta
+[meta]: #meta
+- Name: Add Contributer Graph
+- Start Date: 2020-03-08
+- Author(s): aemengo
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC proposes adding a customized contributer graph to the buildpacks [website](https://buildpacks.io/community/). Please see [mockup](#prior-art) for concept.
+
+# Definitions
+[definitions]: #definitions
+
+* Contribution - Opening a issue, Creating a PR, or Creating an RFC
+* Organization - The corporation represented by those making a contribution
+* OSS community - People engaging in discussion, usage and contribution to an OSS project
+
+# Motivation
+[motivation]: #motivation
+
+Tracking the health of an OSS community can be a very intensive activity. It can consist of keeping tabs on GitHub issues across multiple key repositories, or being an active participant in a multiple OSS slack channels, or keeping up with blog posts, etc. Although these collectively yield the most accurate insight into the health of an OSS community, not everyone can devote the time and attention on a consistent basis.
+
+For these others that are still invested in the health of the community, quantitative metrics are a useful recourse. These "others" could be prospective engineers deciding whether to immerse themselves within the community, or hobbyists looking to champion buildpacks within their organizations.
+
+The expected outcome is that more productive discussions about community engagement can occur when centered around the metrics attempting to depict it.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Please note: _This RFC does not intend to provide something fully realized. Instead, it aims to introduce the concept and have it be shaped by discussion, feasibility, and utility._
+
+At a high level, it is a graph that lives on the buildpacks [website](https://buildpacks.io/community/). The graph will show the full lifecycle of an OSS contribution, which could include the following:
+
+- RFC -> Issue/PR -> Shipped
+- Issue/PR -> Shipped
+
+A contribution is indicated as a data point and linked to other contributions via lines. The graph is seperated into the most critical repositiories of the buildpacks project. Contributions are color-coded by the participants' organization. Contributions are hyperlinked to GitHub issues. The graph aims to show 3 metrics within a given time interval:
+
+- Total amount of contributions
+- Percentage of contributions by organization
```

Comment:
I wonder how important this is at a top-level. I also wonder if it will be confusing or undefined when we get outside contributions. Maybe we can instead annotated each contribution (i.e. make this info a peer of the contributor name)

Reply:
I agree, although it's important to know that some organizations are funding/backing up a project, I think it sort of deters individual contributions if you see that all the changes are dominated by a set of organizations and it might make certain individuals feel that their contributions may not be accepted if they are not associated with these organizations? ( I may be overthinking here)

Reply:
Thanks so much for these words. They go a long way in helping me define how to make this useful.

@samj1912 I especially like what you said about how it might get interpreted. When I have time, my next draft will be more contributor focused.